pkg/controller/plan/migration.go

```diff
@@ -1616,6 +1621,11 @@ func (r *Migration) updatePopulatorCopyProgress(vm *plan.VMStatus, step *plan.St
 			continue
 		}
 
+		_, err = r.isPopulatorFailed(populatorPods, string(pvc.UID))
```

Comment:
why do we need to check it here?

Reply:
but we'd know when the failure happened from the logs of the populator pod, right?
it seems redundant to do this as part of the progress tracking if we anyway track the status of the pod elsewhere

Reply:
based on offline discussions - CDI also check the pod status.
i think it's the best option. but i will try to pass it to the end of the function, if the progress is the same, query if the progress is the same. this might reduce unnecessary queries.